package.json

```diff
@@ -18,7 +20,7 @@
     ],
     "moduleNameMapper": {
       "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
-      "\\.(css|less|sass)$": "<rootDir>/__mocks__/styleMock.js"
+      "\\.(css|less|sass)$": "identity-obj-proxy"
```

Comment:
Took a look at the example code you linked to...to check my understanding: is the idea that `.find("." + icons.iconAudio` is equivalent to `.find(".iconAudio")`, but lets Jest mock the names given by CSS Modules? Doug set up most of this config so I never even looked at the old empty module. 

Will look at the CSS Modules changes more closely tomorrow with @kswenson